helm_chart/ibm-storage-enabler-for-containers-dev/cv-tests/test-01/values.yaml

```diff
@@ -18,7 +18,7 @@ backend: spectrumConnect
 spectrumConnect:
   connectionInfo:
     # IP\FQDN and port of Spectrum Connect server.
-    fqdn: 9.37.37.46
+    fqdn: spectrum-connect.rtp.raleigh.ibm.com
```

Comment:
r u sure its mandatory to state this host here?

Reply:
yes